README.md

```diff
@@ -142,11 +142,29 @@ docker exec -it safe-transaction-service-web-1 python manage.py createsuperuser
 - [v1.3.0 L2](https://github.com/safe-global/safe-deployments/blob/main/src/assets/v1.3.0/gnosis_safe_l2.json)
 - [Other related contracts and previous Safe versions](https://github.com/safe-global/safe-deployments/blob/main/src/assets)
 
-## Troubleshooting
-
-### Issues installing grpc on a Mac M1
-
-If you face issues installing the `grpc` dependency locally (required by this project) on a M1 chip, set `GRPC_PYTHON_BUILD_SYSTEM_OPENSSL=1` and `GRPC_PYTHON_BUILD_SYSTEM_ZLIB=1` and then try to install the dependency again.
+## Service maintenance
+
+Service can run into some issues when running in production:
+
+### Indexing issues
+You can tell there are indexing issues if:
+- Executed transactions are missing from the API (`all-transactions`, `multisig-transactions`, `module-transactions`... endpoints). If you
+use the **web UI** please make sure by checking **Transaction Service API**, as sometimes it can be a cache delay in
+the **Safe Client Gateway** which is between web UI and **Safe Transaction Service**.
```

Comment:
```suggestion
- Executed transactions are missing from the API (`all-transactions`, `multisig-transactions`, `module-transactions`... endpoints). If you use the [Safe{Wallet} Web client](https://github.com/safe-global/safe-wallet-web) you should check what is the current state of the Safe Client Gateway cache as it might have outdated data.
```